pkg/sql/schemachanger/BUILD.bazel

```diff
@@ -99,20 +98,21 @@ sctest_gen(
     ],
 )
 
-sctest_gen(
-    name = "test_mixed_gen",
-    out = "sctest_mixed_generated_test.go",
-    ccl = False,
-    new_cluster_func = "sctest.SingleNodeMixedCluster",
-    package = "schemachanger",
-    test_data = [
-        ":end_to_end_testdata_definitions",
-    ],
-    tests = [
-        "ValidateMixedVersionElements",
-    ],
-    visibility = [
-        ":__pkg__",
-        "//pkg/gen:__pkg__",
-    ],
-)
+# Disabled: #106349
+#sctest_gen(
+#    name = "test_mixed_gen",
+#    out = "sctest_mixed_generated_test.go",
+#    ccl = False,
+#    new_cluster_func = "sctest.SingleNodeMixedCluster",
+#    package = "schemachanger",
+#    test_data = [
+#        ":end_to_end_testdata_definitions",
+#    ],
+#    tests = [
+#        "ValidateMixedVersionElements",
+#    ],
+#    visibility = [
+#        ":__pkg__",
+#        "//pkg/gen:__pkg__",
+#    ],
+#)
```

Comment:
Adding a `skip.WithIssue` call in `ValidateMixedVersionElements` would be preferable from a maintainability point of view (less likely to be forgotten about) and yield the same results.